### ISSUE TITLE
Proposal: do not raise Credo.Check.Refactor.FunctionArity when implementing a behaviour

### ISSUE THREAD

**Issue by amuino** [rrrene/credo#361]
### Environment

* Credo version (`mix credo -v`): 0.7.2
* Erlang/Elixir version (`elixir -v`): 1.4.2
* Operating system: maxOS Sierra (10.12.4) 

### What were you trying to do?
While implementing the `Ecto.Adapter` behaviour, which includes two functions with high arity:
* `execute/6`
* `insert/6`
… I got two "refactoring opportunities" from credo:
```
Function takes too many parameters (arity is 6, max is 5).
```

### Expected outcome
Since I am not able to change the arity of the callbacks defined in the behaviour, I would expect Credo not to issue any warnings.

### Actual outcome
Got one warning for each function.

After trying to ignore them with `@lint`, I got a message suggesting that I open an issue… and here I am  😇

**Comment by rrrene** [comment]
@amuino Thx for reporting this! :+1:
